R/pull.R

```diff
@@ -23,9 +23,9 @@ pulley <- function(resamples, res, col) {
 
   id_cols <- grep("^id", names(resamples), value = TRUE)
   resamples <- dplyr::arrange(resamples, !!!syms(id_cols))
-  pulled_vals <- purrr::map_dfr(res, ~ .x[[col]])
+  pulled_vals <- try(purrr::map_dfr(res, ~ .x[[col]]), silent = TRUE)
```

Comment:
The addition of the `try` here counteracts the deletion of the `try` at line 6, which is needed to surface the errors that occur at line 6 to `.catch_and_log()`.